pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[tool.poetry]
+name = "demisto-py"
+version = "3.3.1beta2"
+description = "\"A Python library for the Demisto SDK\""
```

Comment:
not exactly

Reply:
copy-paste from `setup.py`
` description="A Python library for the Demisto API"`

Reply:
Done